1224-user_administration/1363-administrator_assigns_a_role_to_a_user_part_i.feature

```diff
@@ -0,0 +1,7 @@
+    @1363
+    Scenario: Administrator assigns a Role to a User (Part I)
+        Given that the Administrator searches for User
+        When they click the Add Role button for a User
+        Then the Add Role Dialog is closed
+        And the User Administration Page is opened
+        And they see that the User has got the role
```

Comment:
And they see that the User has <Role>

What is the role?

Reply:
burde kanskje vært eksempler her. Jeg kan legge til det